pkg/minikube/constants/constants.go

```diff
@@ -55,10 +59,24 @@ func MakeMiniPath(fileName ...string) string {
 	return filepath.Join(args...)
 }
 
+// GetProfileHomeDir determines the Minishift home directory for the profile.
+func GetProfileHomeDir() string {
+	if ProfileName != DefaultProfileName {
+		homeEnv, ok := os.LookupEnv(MiniShiftHomeEnv)
+		if ok {
+			return filepath.Join(homeEnv, "profiles", ProfileName)
+		} else {
```

Comment:
Unnecessary else.
We can write `return filepath.Join(util.HomeDir(), ".minishift", "profiles", ProfileName)` after `if`.